qwik-graphql-tailwind/src/components/pagination/pagination.tsx

```diff
@@ -1,24 +1,27 @@
 import { $, component$ } from '@builder.io/qwik';
+import { useNavigate } from '@builder.io/qwik-city';
 
 export interface PaginationProps {
+  tab?: string;
   pageInfo?: any;
```

Comment:
Maybe we can add the pageInfo type here. It should be:

` { 
       hasPreviousPage: boolean;
       hasNextPage: boolean;
       startCursor?: string;
       endCursor?: string ;
    }`

Reply:
cool